.mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
```

Comment:
I tried `ty` (from Astral) but the first two warnings I read in its output were false-postivies, so I didn't bother checking more.